react-challenge.html

```diff
@@ -23,36 +23,40 @@
     // No need to change anything ABOVE this line ^
 
 
-
+//added brackets to quotation marks for current mood
+//added this.state.moods to this.setState section.
       class App extends React.Component{
         constructor(props){
           super(props)
           this.state = {
             moods: ["😃", "😴", "😍", "🥺", "😡", "🤓", "🤪"],
-            currentMood: ""
+            currentMood: [""]
           }
         }
 
         handleMoodOptions = () => {
-          let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          let randomIndex = Math.floor(Math.random() *  this.state.moods.length )
+          this.setState({ currentMood: this.state.moods[randomIndex] })
         }
 
         render(){
+          let { currentMood } = this.state
           return(
 
+              //the <></> were missing to nest multiple elements
+              //was missing this. for the onClick
+              <>
               <h1>Welcome to the Mood Meter App</h1>
 
               <h3>Mood Options:</h3>
 
               <div style={{ fontSize: "40px", height: "50px"}}>
-                { this.state.currentMood }
+                { this.currentMood }
               </div>
-
-              <button onClick={ handleMoodOptions }>
+              <button onClick={ this.handleMoodOptions }>
```

Comment:
yes! because we're referencing a class method we need to properly access it through the keyword `this`